### DISCUSSION TITLE
Adobe AIR (Animate): Could not access the digital cerificate

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3104] [Q&A]
Howdy! When i make publicate my app, i see some error. Serificate new, i made hes 3 weeks ago.
![Uploading Screenshot_1.png…]()

**Comment by raresn** [comment]
And is the certificate still valid? These problems happen all the time. 
-Check the certificate. 
-Make sure you are using the right certificate, maybe you are pointing to the wrong file?

**Reply by ZhorovEgor** [reply]
Mb couse i create certificate on other Mac?

**Reply by ajwfrost** [reply]
The screenshot there hadn't finished uploading before you posted the message, unfortunately .. are you able to edit that and re-upload the screenshot so that we can see what the error is?

If you're using Animate 2024 then it's using a later JRE which means we have to use the "native" code signing on the device, which means if you're using a .p12 file, it also need to be imported into your login certificates in the KeyChain Access app.

thanks

**Reply by ZhorovEgor** [reply]
thx y i regenerated file, all normal work
